### PR TITLE
Added score function into Board class.

### PR DESCRIPTION
**Pull request by Arrinao**
A simple score function that adds 10 points per full_line_clear.

### CODE REVIEW

File: tetris.py [hunk 1]
```diff
@@ -69,14 +69,15 @@ def run_gui():
...
-    small_board = Board(topbar_canvas, 4, 2, D_GREY, (2, 1), None)
+    small_board = Board(topbar_canvas, 4, 2, D_GREY, (2, 1), None, None)
     main_board = Board(
         game_canvas,
         game_width,
         game_height,
         GREY,
         (int(game_width / 2), -2),
         small_board,
+        topbar_score
```
Comment (Akuli) [inline tetris.py:80]: Variable name is not great. How about `topbar_label`?

File: tetris.py [hunk 2]
```diff
@@ -111,8 +112,10 @@ def __init__(self, canvas, width, height, outline_color, current_block_center, s
         self.block_letter = random.choice(block_letters)
         self.rotate_counter = 0
         self.small_board = small_board
+        self.topbar_score = topbar_score
         self.draw_board()
         self.draw_block()
+        self.initial_score = 0
```
Comment (Akuli) [inline tetris.py:118]: Why is this called `initial_score`? It is not always the initial score, it changes when the game is played.

File: tetris.py [hunk 3]
```diff
@@ -292,13 +295,19 @@ def full_line_clear(self):
                     self.landed_blocks[letter] = [(a, b) for (a, b) in coord_list if b > x_line] + [
                         (a, b + 1) for (a, b) in coord_list if b < x_line
                     ]
+                self.score()
+
+    def score(self):
```
Comment (Akuli) [inline tetris.py:300]:
```suggestion
    def increment_score(self):
```

File: tetris.py [hunk 4]
```diff
@@ -292,13 +295,19 @@ def full_line_clear(self):
...
+        self.initial_score += 10
+        self.topbar_score.config(text=self.initial_score)
 
 
 class TetrisGUI:
     def __init__(self, main_board, topbar_time):
         self.main_board = main_board
         self.topbar_time = topbar_time
         self.start_time = time.time()
+        self.initial_score = 0
```
Comment (Akuli) [inline tetris.py:310]: Delete this

**Review by Akuli — CHANGES_REQUESTED** [review]

File: tetris.py (hunk 1 above)
Comment (Arrinao) [reply tetris.py:80]: There are three labels, (actually one of them is canvas) that the topbar frame is divided into. topbar_time, topbar_score and topbar_canvas. Don't you think it's better to keep the name?

File: tetris.py (hunk 2 above)
Comment (Arrinao) [reply tetris.py:118]: Changed to score

Comment (Akuli) [reply tetris.py:118]: doesn't seem like it? please push

Comment (Arrinao) [reply tetris.py]: Done

File: tetris.py (hunk 4 above)
Comment (Arrinao) [reply tetris.py]: Deleted

**Comment by Akuli** [comment]
There are merge conflicts, you need to pull on main and merge main into `score` branch

**Comment by Akuli** [comment]
merged

**Review by Akuli — APPROVED** [review]
a bit weird to change flash colors in a score pr, but whatever :)

**Pull request merged**